pytest.ini

```diff
@@ -1,6 +0,0 @@
-[pytest]
```

Comment:
Why are you deleting our pytest.ini?

Reply:
I thought it was a local file of mine for some reason.